src/content/docs/fr/guides/aliases.mdx

```diff
@@ -1,44 +1,46 @@
 ---
 title: Alias
 description: Une introduction aux alias avec Astro.
+i18nReady: true
 ---
 
 Un **alias** est une façon de créer des raccourcis pour vos imports.
 
 Les alias peuvent aider à améliorer l'expérience de développement dans les codebases avec de nombreux dossiers ou importations relatives.
```

Comment:
I don't know if we shouldn't translate the `codebases` word...

Reply:
I dont know

Reply:
I think is good like this